src/batch/cancel.rst

```diff
@@ -1,32 +1,40 @@
+.. |br| raw:: html
+
+   <br>
+
 Canceling Jobs
 ==============
 You can cancel jobs with the ``scancel`` command.
 
-To cancel a single job you can supply the ``jobid`` as the argument.
+To cancel a single job you can supply the ``jobid`` as the argument:
 
 .. code-block:: console
 
    [testuser@fe-ac-02 ~]$ scancel <jobid>
```

Comment:
If you remove <> everywhere else, you should do the same for `<jobid>` here

Reply:
OK